decidim-selectable-news/app/commands/decidim/admin/deliver_newsletter.rb

```diff
@@ -25,13 +25,13 @@ def call
       private
 
       def send_newsletter!
-        NewsletterJob.perform_later(@newsletter, params[:process_id])
-
-        Decidim::ActionLogger.log(
+        Decidim.traceability.perform_action!(
           "deliver",
-          @user,
-          @newsletter
-        )
+          @newsletter,
```

Comment:
The order of the params has changed, are they correct for `perform_action!`?

Reply:
Yes! it changed! And yes, it's correct now!

Check: https://github.com/decidim/decidim/blob/d94c117fd1dc890ed40084010190971c7d101fed/decidim-admin/app/commands/decidim/admin/deliver_newsletter.rb#L28